Dockerfile

```diff
@@ -0,0 +1,132 @@
+# to build
+# docker build -t centos7_sal --rm=true .
+
+# Clean up
+# docker rm $(docker ps -a -q)
+
+
+FROM centos:7
+
+LABEL name="CentOS Base Image" \
+    vendor="CentOS" \
+    license="GPLv2" \
+    build-date="20161214"
+
+# Definitions
+ARG FITSIO_VERSION=1.0.4
+ARG SAL_VERSION=3.10.0_001
+ARG OSPL_VERSION=6.9.0
+ARG GIT_LSST="https://github.com/lsst-dm"
+ARG INSTALL_PATH=/opt/lsst
+ARG HEADERSERVICE_VERSION=0.9.7
+ARG SALPYTOOLS_VERSION=0.9.7
+ARG HSUSER=felipe
```

Comment:
This should be a "standard" user;  we're using "iip" for all the services in ctrl_iip.   Doesn't have to be "iip", but probably shouldn't be your own either.

Reply:
Thanks, I changed the default to something more formal, like `headerservice`